Source/Bot/Bot.swift

```diff
@@ -69,6 +72,12 @@ protocol Bot {
     func login(network: NetworkKey, hmacKey: HMACKey?, secret: Secret, completion: @escaping ErrorCompletion)
     func logout(completion: @escaping ErrorCompletion)
 
+    // MARK: Invites
+
+    // Redeem uses the invite information and accepts it.
+    // It adds the pub behind the address to the connection sheduling table and follows it.
+    func inviteRedeem(token: String, completion: @escaping ErrorCompletion)
+
```

Comment:
@martindsq Here are the two new functions you need.

Reply:
Got it!